content/docs-v1/zooming-images.md

```diff
@@ -57,21 +57,41 @@ To speed development time, Quire will skip processing zooming images that have p
-- Don't use too many on one page or in one lightbox
+- Don't use too many images on one page or in one lightbox
 - Use placeholder images until ready to build the actual work 
 - Host images externally (see the following section)
 
+## Add Existing External IIIF Images
+
+In addition to creating deep zooming images in your Quire project by adding `zoom: true` to a `figures.yaml` entry, you can also use existing external IIIF images. This is achieved by adding a few additional attributes to the `figures.yaml` entry, including `manifestId`, `canvasId`, and `printImage`. 
+
+```yaml
+- id: "irises"    
+  manifestId: https://media.getty.edu/iiif/manifest/53be857e-41e8-4198-b45d-2e0f52d3051b
+  canvasId: https://media.getty.edu/iiif/manifest/canvas/e5d29650-11f8-4897-9540-54a9dd65b04f.json
+  printImage: figures/irises.jpg
+  src: figures/irises.jpg 
+```
+
+The `manifestId` or manifest URL is a link to a JSON file that contains important metadata about an object including things like artist, name, and copyright information. A manifest URL will always accompany a IIIF image, however, each institution displays this information a bit differently. Sometimes there is a IIIF button that will take you to a manifest, other institutions might include the manifest URL below or alongside the tombstone information on an object's collection page. Here is an example of where the manifest URL is found on the Getty collection page for Van Gogh's *Irises*:
+
+{{< q-figure id="iiif-manifest" >}}
+
+The next attribute you need to include in the `figures.yaml` is the `canvasId`. Every institution's manifest is formatted differently so locating the `canvasID` can be a bit tricky. The easiest way to find it is with a tool like Liz Fischer's [IIIF Manifest Explorer](https://www.lizmfischer.com/iiif-tools/manifest). If you plug the manifest URL (`manifestId`) into the URL bar, the tool will breakdown all the metadata in the manifest so it is easier to read, including identifying the `canvasID` which you will then add to your `figures.yaml` entry.
+
+{{< q-figure id="liz-fischer" >}}
 
-## Use Existing External IIIF Images
+In the `figures.yaml` you will also need to include a `printImage` attribute and a `src` attribute. These point to where you have saved your image in your project. So for the example above, `irises.jpg` is saved in the projects `_assets/images/figures` folder. The `src` value will find the image and add it to your Markdown page and the `printImage` will make sure that image is included in the PDF/print version of your publication. 
 
-Coming Soon!
\ No newline at end of file
+{{< q-class "box tip" >}}
+- At the moment the +/- signs typically used to zoom in and out of an image are not available in Quire. Instead zooming can be achieved by scrolling or pinching. Adding these controls back to Quire is part of the work laid out in [GitHub issue #815](https://github.com/thegetty/quire/issues/815) mentioned above.
```

Comment:
Positioned here, this feels like it's related to external IIIF images specifically. Should it maybe move up to the first section of the page?